index.js

```diff
@@ -1,285 +0,0 @@
-    [{ground: randomMaterial(),x:0,y:950},{ground: randomMaterial(),x:50,y:950},{ground: randomMaterial(),x:100,y:950},{ground: randomMaterial(),x:150,y:950},{ground: randomMaterial(),x:200,y:950}, {ground: randomMaterial(),x:250,y:950},  {ground: randomMaterial(),x:300,y:950},  {ground: randomMaterial(),x:350,y:950},  {ground: randomMaterial(),x:400,y:950},  {ground: randomMaterial(),x:450,y:950},  {ground: randomMaterial(),x:500,y:950},{ground: randomMaterial(),x:550,y:950},  {ground: randomMaterial(),x:600,y:950}],
-    [{ground: randomMaterial(),x:0,y:1000},{ground: randomMaterial(),x:50,y:1000},{ground: randomMaterial(),x:100,y:1000},{ground: randomMaterial(),x:150,y:1000},{ground: randomMaterial(),x:200,y:1000}, {ground: randomMaterial(),x:250,y:1000},  {ground: randomMaterial(),x:300,y:1000},  {ground: randomMaterial(),x:350,y:1000},  {ground: randomMaterial(),x:400,y:1000},  {ground: randomMaterial(),x:450,y:1000},  {ground: randomMaterial(),x:500,y:1000},{ground: randomMaterial(),x:550,y:1000},  {ground: randomMaterial(),x:600,y:1000}],
-    [{ground: rock,x:0,y:1050},{ground: rock,x:50,y:1050},{ground: rock,x:100,y:1050},{ground: rock,x:150,y:1050},{ground: rock,x:200,y:1050}, {ground: rock,x:250,y:1050},  {ground: rock,x:300,y:1050},  {ground: rock,x:350,y:1050},  {ground: rock,x:400,y:1050},  {ground: rock,x:450,y:1050},  {ground: rock,x:500,y:1050},{ground: rock,x:550,y:1050},  {ground: rock,x:600,y:1050}],
-    [{ground: gold,x:0,y:1100},{ground: gold,x:50,y:1100},{ground: gold,x:100,y:1100},{ground: gold,x:150,y:1100},{ground: gold,x:200,y:1100}, {ground: gold,x:250,y:1100},  {ground: gold,x:300,y:1100},  {ground: gold,x:350,y:1100},  {ground: gold,x:400,y:1100},  {ground: gold,x:450,y:1100},  {ground: gold,x:500,y:1100},{ground: gold,x:550,y:1100},  {ground: gold,x:600,y:1100}]
-
-]
-}
-*/
-function makeMap() {
-    // random map, but I want first 3-4 layers to be empty and bottom two to be rock and gold.
-    //to make the first layers empty, change the 0 in the second forloop to be j= 3, then it starts at the third layer
-    const row = 13;
-    const col = 25;
-    mapArr = [];
-  
-    for (let i = 1; i < col; i++) {
-       if(!i==0){
-            mapArr.push([]) 
-        }
-      mapArr.push([]);
-      
-      if (i<col-2){
-      for (let j = 0; j < row; j++) { 
-        let tempObj = {ground: randomMaterial(),x:j*50 ,y:i*50};
-        mapArr[i].push(tempObj); }
-   
-     }else if (i == col-2){
-        for (let g = 0; g < row; g++) { 
-            let tempObj = {ground: rock,x:g*50 ,y:i*50};
-            mapArr[i].push(tempObj);}
-     }else{ 
-        for (let k = 0; k < row; k++) { 
-            let tempObj = {ground: gold,x:k*50 ,y:i*50};
-            mapArr[i].push(tempObj);
-            
-        }
```

Comment:
also it seems this is not properly formatted. Please run your code through prettier and setup auto-saving with prettier in vscode :) makes all the code you write auto-formatted as soon as you save.